dali/operators/image/convolution/laplacian.h

```diff
@@ -29,13 +29,20 @@ namespace dali {
 #define LAPLACIAN_CPU_SUPPORTED_TYPES \
   (uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t, float16, float)
 
+// TODO(klecki): float16 support - it's not easily compatible with float window,
+// need to introduce some cast in between and expose it in the kernels
+// #define LAPLACIAN_GPU_SUPPORTED_TYPES (float)
```

Comment:
I purposely copied that part as it seems to apply here as well as the op is based on the gpu convolution. I thought it would be easier to navigate, but maybe it is unnecessary or irrelevant?

Reply:
Just the `// #define LAPLACIAN_GPU_SUPPORTED_TYPES (float)` part

Reply:
done